algorithms/linfa-linear/benches/ols_bench.rs

```diff
@@ -0,0 +1,28 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use linfa::traits::Fit;
+use linfa_datasets::generate::make_dataset;
+use linfa_linear::LinearRegression;
+use statrs::distribution::{DiscreteUniform, Laplace};
+
+fn perform_ols(num_rows: usize) {
+    let feat_distr = Laplace::new(0.5, 5.).unwrap();
+    let target_distr = DiscreteUniform::new(0, 5).unwrap();
+    let num_feats: usize = 5;
+    let dataset = make_dataset(num_rows, num_feats, feat_distr, target_distr);
```

Comment:
Move `make_dataset` outside of this function so that it's not included in the benchmark times.

Reply:
gotcha.